docs/virtual_machines/interfaces_and_networks.md

```diff
@@ -595,6 +595,48 @@ has no DHCP server at all. Therefore, the VM won't have the search domains infor
 reaching a destination using its FQDN is not possible.
 Tracking issue - https://github.com/kubevirt/kubevirt/issues/7184
 
+### passt
+
+`passt` is a new approach for user-mode network that can be used as a simple replacement for Slirp.
+
+```yaml
+kind: VM
+spec:
+  domain:
+    devices:
+      interfaces:
+        - name: red
+          passt: {} # connect using passt mode
+  networks:
+  - name: red
+    pod: {}
+```
+
+#### Requirements:
+1. To get better performance the node should be configured with:
+```
+sysctl -w net.core.rmem_max = 33554432
+sysctl -w net.core.wmem_max = 33554432
+```
+2. To run multiple passt VMs with no explicit ports, the node's `fs.file_max` should be increased
+   (at least ~2^18 per passt with no explicit ports VM):
+```
+sysctl -w fs.file_max = 9223372036854775807
+```
+3. The `vmi.Spec.Domain.Resources.Requests.Memory` of a passt VM with no explicit ports should be at least 2048M.
```

Comment:
Is this at all related to spec.domain.memory.guest? If I set both to the same value, all the memory would be given to the guest, and you would not have enough left for passt running in launcher?

Reply:
hm, afaiu, virt-controller calculates the launcher overhead (some 100MB, depending on the "requested" guest features) and adds those to what you request here - the intention being assuring the guest can operate properly (while setting this threshold at the pod level).

Reply:
I believe Miguel is right and the memory requested from kubevirt when running a VM is global (not pod part and guest part).
`spec.domain.memory.guest` is used for [over-commitment](https://kubevirt.io/user-guide/operations/node_overcommit/)